file/file.h

```diff
@@ -52,7 +52,24 @@ namespace bricks {
 // Platform-indepenent, injection-friendly filesystem wrapper.
 // TODO(dkorolev): Move the above methods under FileSystem.
 struct FileSystem {
-  static inline std::string ReadFileAsString(std::string const& file_name) {
+  static inline std::string GetFileExtension(const std::string& file_name) {
+    const static auto get_extension_from_basename = [](const std::string& fn) {
+      const size_t i = fn.find_last_of('.');
+      return i == std::string::npos ? "" : fn.substr(i + 1);
+    };  // LCOV_EXCL_LINE
+
+    static_assert(std::string::npos + 1 == 0, "This invariant is used in the next line.");
```

Comment:
Yes, it's guaranteed to be the case. The ```static_assert``` is more of a comment for the reader of the code.